system-tests/test/deprecated_spec.ts

```diff
@@ -64,10 +74,12 @@ describe('deprecated before:browser:launch args', () => {
     project: beforeBrowserLaunchProject,
     spec: 'app_spec.js,app_spec2.js',
     snapshot: true,
-    stdoutInclude: 'Deprecation Warning:',
+    onStdout: includesString('Deprecation Warning:'),
   })
 
-  systemTests.it('no mutate return', {
+  // TODO: fix/remove this test, it should be warning but is not
+  // https://github.com/cypress-io/cypress/issues/20436
+  systemTests.it.skip('no mutate return', {
```

Comment:
I'm a little confused, is this test incorrect validating this is throwing a warning for this message or is this not actually warning like we expected it to?

I know the linked issue is to remove the warning, but I'm wondering if should be linked next to that part of the code-base what added the warning

Reply:
It should be warning, but it is not warning. This test is not actually asserting that this is warning (check the snapshots). It was erroneously passing before because `stdoutInclude` and `stdoutExclude` never actually did anything. This was missed on the review for the deprecation when it went into 4.0.0.

Reply:
I feel like I'm missing something obvious, If it's not warning, why would it a breaking change to remove it? Or are you saying we need to add the warning so we can remove the deprecated `before:browser:launch` usage?

Reply:
#20436 is a breaking change not because it removes the warnings, but because it removes the massaging that we do that enables users to use the outdated `before:browser:launch` pattern. So users who are receiving warnings today will receive errors once #20436 is done.